swssconfig/swssconfig.cpp

```diff
@@ -21,9 +23,20 @@ const int el_count = 2;
 
 const string SWSS_CONFIG_DIR    = "/etc/swss/config.d/";
 
+/*
+ * swssconfig will only connect to local orchagent ZMQ endpoint.
+ */
+const char* ZMQ_LOCAL_ADDRESS = "tcp://localhost";
+
+const char* DASH_TABLE_PREFIX = "DASH_";
+
+/* ZMQ enable feature flag for orch events */
+const int ENABLE_ZMQ_DASH  = 0x01;
+const int ENABLE_ZMQ_ROUTE = 0x02;
+
 void usage()
 {
-    cout << "Usage: swssconfig [FILE...]" << endl;
+    cout << "Usage: swssconfig [-z zmq_flag] [FILE...]" << endl;
     cout << "       (default config folder is /etc/swss/config.d/)" << endl;
```

Comment:
>folder [](http://example.com/codeflow?start=36&length=6)

Could you add explanation of zmq_flag in the Usage text?

Reply:
Removed because using config_db